app/assets/javascripts/index/element.js

```diff
@@ -1,10 +1,18 @@
 (function () {
+  let abortController = null;
+  const langs = [...new Set([...OSM.preferred_languages.map(l => l.toLowerCase()), "mul", "en"])];
```

Comment:
Can we give this a more specific name than `langs` to avoid guessing what it actually means? Is it *preferred languages from mediawiki point of view*?